en/02_Developer_Guides/06_Testing/04_Fixtures.md

```diff
@@ -176,6 +169,9 @@ App\Test\Team:
     Players: =>App\Test\Player.joe,=>App\Test\Player.jack
 ```
 
+> [!WARNING]
+> Be aware the target of a relationship must be defined before it is referenced, for example the `hurricanes` team must appear in the fixture file before the line `Team: =>App\Test\Team.hurricanes`.
```

Comment:
Moved this from above to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.

Also intentionally converted from hint to warning, because the content is warning about something.